data/campaigns/Eastern_Invasion/scenarios/99_Empire.cfg

```diff
@@ -0,0 +1,1514 @@
+        [/then]
+    [/if]
+    [if]{VARIABLE_CONDITIONAL side equals 4}
+        [then]
+            {VARIABLE taunt_message _"What happened to dying slow? So much for him."                     }
+        [/then]
+    [/if]
+    [if]{VARIABLE_CONDITIONAL side equals 5}
+        [then]
+            {VARIABLE taunt_message _"They really thought they could get away? Haw..."                   }
+        [/then]
+    [/if]
+    [message]
+        speaker=Chief Dra-Nak
+        message=$taunt_message
+    [/message]
+
+    [if]
+        [have_unit]
+            side,count={SIDE},1-99
+        [/have_unit]
+
+        [have_unit]
+            side=2,3,4
+            canrecruit=yes # only trigger this if it's NOT the final leader
+        [/have_unit]
+
+        [then]
+            # joinme message
+            [if]{VARIABLE_CONDITIONAL side equals 2}
+                [then]
+                    {VARIABLE joinme_message _"He called himself your leader, but now you see him for the weakling he really was. You can either join him or fight alongside me and pray I show you mercy."}
+                [/then]
+            [/if]
+            [if]{VARIABLE_CONDITIONAL side equals 3}
+                [then]
+                    {VARIABLE joinme_message _"As for the rest of you - pledge yourselves to me, or die a slow painful death!"}
+                [/then]
+            [/if]
+            [if]{VARIABLE_CONDITIONAL side equals 4}
+                [then]
+                    {VARIABLE joinme_message _"You might as well give it up! Resistance is pointless."}
+                [/then]
+            [/if]
+            [if]{VARIABLE_CONDITIONAL side equals 5}
+                [then]
+                    {VARIABLE joinme_message _"Nobody escapes me. Join me and I’ll treat you fairly, resist me and you’ll die painfully."}
+                [/then]
+            [/if]
+            [message]
+                speaker=Chief Dra-Nak
+                message=$joinme_message
+            [/message]
+
+            # defector message
+            [if]{VARIABLE_CONDITIONAL side equals 2}
+                [then]
+                    {VARIABLE defector_message _"Mortic was strong.
+Alone I am weak.
+I will serve you."}
+                [/then]
+            [/if]
+            [if]{VARIABLE_CONDITIONAL side equals 3}
+                [then]
+                    {VARIABLE defector_message _"I crave not death.
+I will myself to you."}
+                [/then]
+            [/if]
+            [if]{VARIABLE_CONDITIONAL side equals 4}
+                [then]
+                    {VARIABLE defector_message _"These orcs enrage me,
+But I possess no strength to resist.
+I will serve you."}
+                [/then]
+            [/if]
+            [if]{VARIABLE_CONDITIONAL side equals 5}
+                [then]
+                    {VARIABLE defector_message _"Ok, ok, just please don’t hurt me!"}
```

Comment:
'Okay'

Reply:
Again, I'd replace it with something completely different instead.